### PR TITLE
[hu] Machine translation of missing keys

### PR DESCRIPTION
**Pull request by andrey-helldar**

### CODE REVIEW

**Comment by caouecs** [comment]
@veencel can you help us and review this automatic pull request, please ? thanks by advance

**Comment by szepeviktor** [comment]
> Machine translation of

That must be a well oiled machine!
These are correct in Hungarian.

**Comment by szepeviktor** [comment]
BTW **Sub-arrays** are not translated.

**Comment by caouecs** [comment]
> BTW **Sub-arrays** are not translated.

Can you help us to translate sub-array tags ? Thanks by advance

File: locales/hu/validation-inline.php [hunk 1]
```diff
@@ -82,46 +82,46 @@
...
+    'prohibited_unless'    => 'Ez a mező tilos, kivéve, ha a :other :values-ben van.',
+    'regex'                => 'Ez a formátum érvénytelen.',
+    'relatable'            => 'Lehet, hogy ez a mező nem kapcsolódik ehhez az erőforráshoz.',
+    'required'             => 'Ez a mező szükséges.',
+    'required_if'          => 'Erre a mezőre akkor van szükség, ha a :other :value.',
+    'required_unless'      => 'Erre a mezőre csak akkor van szükség, ha a :other :values-ban van.',
+    'required_with'        => 'Ez a mező akkor szükséges, ha :values van jelen.',
+    'required_with_all'    => 'Ez a mező akkor szükséges, ha :values van jelen.',
+    'required_without'     => 'Ez a mező akkor szükséges, ha :values nincs jelen.',
+    'required_without_all' => 'Erre a mezőre akkor van szükség, ha a :values egyik sem jelen van.',
```
Comment (szepeviktor) [inline locales/hu/validation-inline.php:110]:
```suggestion
    'required_without_all' => 'Erre a mezőre akkor van szükség, ha egyik :values sincs jelen.',
```

File: locales/hu/validation-inline.php [hunk 2]
```diff
@@ -56,14 +56,14 @@
...
-    'ipv4'                 => 'This must be a valid IPv4 address.',
-    'ipv6'                 => 'This must be a valid IPv6 address.',
-    'json'                 => 'This must be a valid JSON string.',
+    'image'                => 'Ez egy kép lehet.',
+    'in'                   => 'A kiválasztott érték érvénytelen.',
+    'in_array'             => 'Ez az érték nem létezik :other-ban.',
+    'integer'              => 'Ennek egész számnak kell lennie.',
+    'ip'                   => 'Ennek érvényes IP-címnek kell lennie.',
+    'ipv4'                 => 'Ennek érvényes IPv4-címnek kell lennie.',
+    'ipv6'                 => 'Ennek érvényes IPv6-címnek kell lennie.',
```
Comment (szepeviktor) [inline locales/hu/validation-inline.php:65]:
```suggestion
    'ipv4'                 => 'Ennek érvényes IPv4 címnek kell lennie.',
    'ipv6'                 => 'Ennek érvényes IPv6 címnek kell lennie.',
```

File: locales/hu/validation-inline.php [hunk 3]
```diff
@@ -12,38 +12,38 @@
...
+    'different'            => 'Ennek az értéknek különböznie kell :other-től.',
+    'digits'               => 'Ennek :digits számjegynek kell lennie.',
+    'digits_between'       => 'Ennek :min és :max között kell lennie.',
+    'dimensions'           => 'Ez a kép érvénytelen méretekkel rendelkezik.',
+    'distinct'             => 'Ez a mező ismétlődő értékkel rendelkezik.',
+    'email'                => 'Ennek érvényes e-mail címnek kell lennie.',
+    'ends_with'            => 'Ennek az alábbiak egyikével kell végződnie: :values.',
+    'exists'               => 'A kiválasztott érték érvénytelen.',
+    'file'                 => 'A tartalomnak fájlnak kell lennie.',
+    'filled'               => 'Ennek a mezőnek értéknek kell lennie.',
```
Comment (szepeviktor) [inline locales/hu/validation-inline.php:46]:
```suggestion
    'filled'               => 'Ennek a mezőnek kell legyen értéke.',
```

**Comment by caouecs** [comment]
ok, thank you, I think at a pull request ;)

I do fixes

**Comment by caouecs** [comment]
done

File: locales/hu/hu.json [hunk 4]
```diff
@@ -1,617 +1,617 @@
...
-    "Cuba": "Cuba",
-    "Curaçao": "Curaçao",
-    "Current Password": "Current Password",
-    "Customize": "Customize",
-    "Cyprus": "Cyprus",
-    "Czech Republic": "Czechia",
-    "Dashboard": "Dashboard",
+    "could not be found.": "nem találtam.",
+    "Create": "Létrehozás",
+    "Create & Add Another": "Hozzon Létre Egy Másik",
```
Comment (szepeviktor) [inline locales/hu/hu.json:134]:
```suggestion
    "Create & Add Another": "Létrehozás és Egy Másik Hozzáadása",
```

File: locales/hu/hu.json [hunk 5]
```diff
@@ -1,617 +1,617 @@
...
-    "Log in": "Log in",
-    "Log out": "Log out",
-    "Log Out": "Log Out",
-    "Log Out Other Browser Sessions": "Log Out Other Browser Sessions",
+    "Lithuania": "Litvánia",
+    "Load :perPage More": "Betöltése :perpage több",
+    "Log in": "Bejelentkezés",
+    "Log out": "Kijelentkezés",
+    "Log Out": "Kijelentkezés",
+    "Log Out Other Browser Sessions": "Jelentkezzen Ki Más Böngésző Munkamenetek",
```
Comment (szepeviktor) [inline locales/hu/hu.json:293]:
```suggestion
    "Log Out Other Browser Sessions": "Jelentkezzen Ki A Többi Böngésző Munkamenetből",
```

File: locales/hu/hu.json [hunk 6]
```diff
@@ -1,617 +1,617 @@
...
     "Login": "Bejelentkezés",
     "Logout": "Kijelentkezés",
-    "Logout Other Browser Sessions": "Logout Other Browser Sessions",
-    "Luxembourg": "Luxembourg",
-    "Macao": "Macao",
-    "Macedonia": "North Macedonia",
-    "Madagascar": "Madagascar",
+    "Logout Other Browser Sessions": "Kijelentkezés Egyéb Böngészőszakaszok",
```
Comment (szepeviktor) [inline locales/hu/hu.json:296]:
```suggestion
    "Logout Other Browser Sessions": "Jelentkezzen Ki A Többi Böngésző Munkamenetből",
```

**Comment by caouecs** [comment]
done

**Comment by szepeviktor** [comment]
> Can you help us to translate sub-array tags ? Thanks by advance

I think machine translation does more than me.

**Comment by caouecs** [comment]
> > Can you help us to translate sub-array tags ? Thanks by advance
> 
> I think machine translation does more than me.

a human is better than a machine ;)

**Comment by szepeviktor** [comment]
I lack an API!

**Pull request merged**